src/pint/models/pulsar_binary.py

```diff
@@ -390,7 +389,7 @@ def update_binary_object(self, toas, acc_delay=None):
                     PulsarEcliptic(ecl=self._parent.ECL.value)
                 ).cartesian.xyz.transpose()
                 updates["psr_pos"] = self._parent.ssb_to_psb_xyz_ECL(
-                    epoch=tbl["tdbld"].astype(np.float64), ecl=self._parent.ECL.value
+                    epoch=tbl["tdbld"], ecl=self._parent.ECL.value
```

Comment:
Should this be the ld version, or the float version?

Reply:
changed.